core/server/common/pom.xml

```diff
@@ -42,6 +42,14 @@
       <groupId>io.atomix.copycat.alluxio</groupId>
       <artifactId>copycat-client</artifactId>
```

Comment:
When can we get rid of these copycat dependencies?

Reply:
Once we removes all copycat usages in transport and backup restore functions then we can remove it.